tests/microos/verify_setup.pm

```diff
@@ -300,6 +304,15 @@ sub run {
     file_tests();
     disk_tests();
 
+    if (get_var('FIRST_BOOT_CONFIG') =~ /combustion/ && !(is_sle_micro('<5.5') || is_leap_micro('<5.5'))) {
+        test_combustion_prepare();
+
+        if (script_run('command -v wicked &> /dev/null') == 0 && is_jeos && (is_sle('15-SP6+') || is_leap('15.6+'))) {
```

Comment:
IMO the version checks here are superfluous. This code is only reached if combustion with prepare support is used, so only the check for wicked is needed.

Reply:
I think that sle-micro 5.1 still has wicked, so technically only jeos on sle/leap requires this check

Reply:
5.1 doesn't support prepare

Reply:
Updated